uniffi_bindgen/src/interface/types/mod.rs

```diff
@@ -199,7 +202,10 @@ impl From<&Type> for FfiType {
                 name,
                 kind: ExternalKind::Interface,
                 ..
-            } => FfiType::RustArcPtr(name.clone()),
+            } => FfiType::RustArcPtr {
+                inner: name.clone(),
+                owned: false,
```

Comment:
I'm hoping that that this never needs to be owned. Please correct me if I'm wrong, because I'm not very familiar with the use cases of these type conversions.